src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -599,8 +595,13 @@
     <Field Name="Checked" Type="bool"/>
     <Field Name="ExplicitCastInCode" Type="bool"/>
     <Field Name="ConstantValueOpt" Type="ConstantValue" Null="allow"/>
-    <!-- True for explicit casts with top-level `?`. Used by NullableWalker. -->
-    <Field Name="IsExplicitlyNullable" Type="bool" Null="allow"/>
+     <!--
+         Reference to a group instance common to all BoundConversions from a single
```

Comment:
>all BoundConversions [](start = 49, length = 20)

I am assuming this is about chained BoundConversions that together represent a single conversion. It would be good to make this clear in the comment, perhaps even with an example.